### DISCUSSION TITLE
Caching Data / Force deletion of local directory

### DISCUSSION THREAD

**Discussion by dbauszus-glx** [vercel/vercel#5207] [Help]
Can we get more information in regard to the caching of data?

The documentation in regards to developing our own runtime mentions the -f (--force) global option which will force the deletion of the local directory.

_The cache will be invalidated if any of those items changes. The user can bypass the cache by running vercel -f._

https://vercel.com/docs/cli#options/global-options/force

Is it possible to use the force through the vercel web interface?

**Comment by paulogdm** [comment]
The "redeploy" button will also clear the cache. Git it a try and let us know.

**Reply by dbauszus-glx** [reply]
@paulogdm unfortunately our deployments are from local repositories and not linked to a GitHub repo at the moment. Would it be possible to have the option to force the deletion of local directory for any deployment?

**Reply by paulogdm** [reply]
You can use `vercel --force` if you need to clear the cache when deploying locally. Your framework can also give options to clear the cache (`gatsby clean`, for example).

**Reply by dbauszus-glx** [reply]
Yes. we are doing this at the moment. We are not using any framework.
